src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -274,29 +273,40 @@ class AbstractIntegrationDetailedView<
 
   //Returns the information about the integration description and features
   renderInformationCard() {
-    const {FeatureList} = getIntegrationFeatureGate();
+    const {IntegrationDirectoryFeatureList} = getIntegrationFeatureGate();
 
     return (
       <React.Fragment>
-        <Description dangerouslySetInnerHTML={{__html: marked(this.description)}} />
-        <FeatureList {...this.featureProps} provider={{key: this.integrationSlug}} />
-        {this.renderPermissions()}
-        <Metadata>
-          {!!this.author && <AuthorName>{t('By %s', this.author)}</AuthorName>}
-          <div>
+        <div style={{display: 'flex'}}>
```

Comment:
@NisanthanNanthakumar When you need to add inline styles like this you should use a styled component instead